### PR TITLE
RENAME appCode to projectToken

### PR DESCRIPTION
**Pull request by ndelangen**
https://github.com/chromaui/chromatic/issues/3285

do not merge yet, must wait for chromatic-cli package to be released and then updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ it does NOT:
           yarn && yarn build
       - uses: chromaui/action@v1
         with: 
-          appCode: <insert the chromatic appToken here>
+          projectToken: <insert the chromatic appToken here>
```
Comment (tmeasday) [inline README.md:32]:
```suggestion
          projectToken: <insert the chromatic projectToken here>
```

File: README.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ it does NOT:
 - uses: chromaui/action@v1
   with:
     token: ${{ secrets.GITHUB_TOKEN }}
-    appCode: 'Your chromatic/chroma app_code'
+    projectToken: 'Your chromatic/chroma app_code'
```
Comment (tmeasday) [inline README.md:42]:
```suggestion
    projectToken: 'Your chromatic/chroma project token'
```

File: README.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ We suggest you use a secret to hide to app-code:
 - uses: chromaui/action@v1
   with:
     token: ${{ secrets.GITHUB_TOKEN }}
-    appCode: ${{ secrets.CHROMATIC_APP_CODE }}
+    projectToken: ${{ secrets.CHROMATIC_APP_CODE }}
```
Comment (tmeasday) [inline README.md:58]: I think we should change the env var too?

File: action.yml [hunk 4]
```diff
@@ -3,7 +3,7 @@ author: Chroma
 description: "Publish & test your storybook with chromatic"
 
 inputs: 
-  appCode: 
+  projectToken: 
     description: "Your chromatic/chroma app_code"
```
Comment (tmeasday) [inline action.yml:7]:
```suggestion
    description: "Your chromatic/chroma project token”
```

File: README.md (hunk 3 above)
Comment (ndelangen) [reply README.md:58]: true

File: README.md [hunk 5]
```diff
@@ -55,7 +55,7 @@ We suggest you use a secret to hide to app-code:
 - uses: chromaui/action@v1
   with:
     token: ${{ secrets.GITHUB_TOKEN }}
-    projectToken: ${{ secrets.CHROMATIC_APP_CODE }}
+    projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
```
Comment (tmeasday) [inline README.md:58]: Do you think we need back-compat here?

Comment (ndelangen) [reply README.md:58]: We do not, this API isn't changing. It's just the example that I changed.

File: README.md [hunk 6]
```diff
@@ -39,7 +39,7 @@ it does NOT:
 - uses: chromaui/action@v1
   with:
     token: ${{ secrets.GITHUB_TOKEN }}
-    appCode: 'Your chromatic/chroma app_code'
+    projectToken: 'Your chromatic/chroma project token'
```
Comment (zol) [inline README.md:42]: @ndelangen remove 'chroma' from this line

**Comment by zol** [comment]
@ndelangen I made some tweaks to the README so be careful when merging this as you'll have to manually resolve a couple of (trivial) conflicts.

**Comment by ndelangen** [comment]
I think i resolved it @zol

**Pull request merged**